app/src/main/java/com/marvel/marvel/main/model/ApiRetrofit.kt

```diff
@@ -0,0 +1,41 @@
+package com.marvel.marvel.main.model
+
+import com.marvel.marvel.main.model.pojos.Comics
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+import rx.Observable
+import java.util.concurrent.TimeUnit
+
+
+class ApiRetrofit() : Api {
+    companion object {
+        private const val MAIN_URL = "https://gateway.marvel.com:443/"
+        private const val TIMEOUT_IN_SECONDS = 60L
+    }
+    private val api: Api
```

Comment:
Why do you use init block? Check out this:
https://github.com/MarcinMoskala/MarvelGallery/blob/master/app/src/main/java/com/sample/marvelgallery/data/network/providers/Retrofit.kt